### DISCUSSION TITLE
Complete NOOB looking for a little help!

### DISCUSSION THREAD

**Discussion by SnakeEaterBand** [alf45tar/PedalinoMini#316] [Q&A]
Hallo team and especially alf45tar,
This project is amazing. I have been searching for a DIY solution exactly like this to control my Quad Cortex. It is super impressive, especially the software side. I am very new at electronics and don't fully understand the build. I realize a lot is learned from doing, but my question is  - is there a more comprehensive step-by-step wiring video/document available? I would happily pay for access to such resource, as I'm sure many newbies like me would. Thank you for the inspiration!

**Comment by jumperrg** [comment]
Hi, I've started to build it a couple of month ago and faced several complications. And If I finish it I want to make a video instructions how to build it, and maybe text/web documentation. I'm not a noob in hardware and software but it is hard for me to build it. Maybe it's because I make it with my custom modifications, but still...

**Comment by alf45tar** [comment]
Every contribution is welcome
